<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shapes</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>371</width>
      <height>371</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="firstParametrLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Выберите фигуру</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="thirdParametrLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" rowspan="2">
        <widget class="QLineEdit" name="secondParametrLineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="figureComboBox"/>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="squareLabel">
         <property name="text">
          <string>S = </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="secondParametrLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLineEdit" name="thirdParametrLineEdit">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="firstParametrLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="4" column="1" rowspan="2">
        <widget class="QPushButton" name="calculatePushButton">
         <property name="text">
          <string>Расчитать</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" rowspan="2">
        <widget class="QPushButton" name="changeFigurePushButton">
         <property name="text">
          <string>Выбрать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="formulaLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>452</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="AuthorMenu">
    <property name="title">
     <string>Авторы</string>
    </property>
    <addaction name="AuthorAction"/>
   </widget>
   <widget class="QMenu" name="ClearMenu">
    <property name="title">
     <string>Очистить</string>
    </property>
    <addaction name="ClearAction"/>
   </widget>
   <widget class="QMenu" name="QuitMenu">
    <property name="title">
     <string>Выход</string>
    </property>
    <addaction name="QuitAction"/>
   </widget>
   <addaction name="AuthorMenu"/>
   <addaction name="ClearMenu"/>
   <addaction name="QuitMenu"/>
  </widget>
  <action name="AuthorAction">
   <property name="text">
    <string>Дмитриев Егор ИП-213</string>
   </property>
  </action>
  <action name="ClearAction">
   <property name="text">
    <string>Очистить ввод</string>
   </property>
  </action>
  <action name="QuitAction">
   <property name="text">
    <string>Выход из приложения</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>QuitAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
